data/siddk.rb

```diff
@@ -692,15 +692,15 @@
 
 entry!('bigham2010vizwiz',
   uist(2010),
-  title('Vizwiz: nearly real-time answers to visual questions'),
+  title('{VizWiz}: nearly real-time answers to visual questions'),
   author('Jeffrey P Bigham and Chandrika Jayant and Hanjie Ji and Greg Little and Andrew Miller and Robert C Miller and Robin Miller and Aubrey Tatarowicz and Brandyn White and Samual White and Tom Yeh'),
   pages(333, 342)
 )
 
 entry!('houlsby2011bayesian',
   arxiv(2011, '1112.5745'),
   title('{B}ayesian active learning for classification and preference learning'),
-  author('Neil Houlsby and Ferenc Husz{\'a}r and Zoubin Ghahramani and M{\'a}t{\'e} Lengyel')
+  author('Neil Houlsby and Ferenc Husz{\\\'a}r and Zoubin Ghahramani and M{\\\'a}t{\\\'e} Lengyel')
```

Comment:
You can also just write á directly if you want (just have to make sure unicode is enabled).